assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * Widgets API defaults
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { CONTEXT_DASHBOARD, CONTEXT_PAGE_DASHBOARD } from './default-contexts';
+import {
+	AREA_DASHBOARD_ALL_TRAFFIC,
+	AREA_DASHBOARD_SEARCH_FUNNEL,
+	AREA_DASHBOARD_POPULARITY,
+	AREA_DASHBOARD_SPEED,
+	AREA_DASHBOARD_EARNINGS,
+	AREA_PAGE_DASHBOARD_SEARCH_FUNNEL,
+	AREA_PAGE_DASHBOARD_ALL_TRAFFIC,
+	AREA_PAGE_DASHBOARD_TOP_QUERIES,
+	AREA_PAGE_DASHBOARD_SPEED,
+} from './default-areas';
+
+/**
+ * Defines default widget areas for a given context.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} widgetsAPI Widgets API.
+ */
+export function registerDefaults( widgetsAPI ) {
+	widgetsAPI.registerWidgetArea( AREA_DASHBOARD_ALL_TRAFFIC, {
+		title: __( 'All Traffic', 'google-site-kit' ),
+		subtitle: __( 'How people found your site.', 'google-site-kit' ),
+		priority: 1,
```

Comment:
@felixarntz would it be better to use priorities multiple of ten for the core widget areas? Like 10, 20, 30 and so on, instead of 1, 2, 3... In this case if 3P wants to insert a new area between 20 an 30, they can use 25th priority.

Reply:
@eugene-manuilov I thought about this, but the idea here is that for now our core widget areas should be higher priority than anything else, at least by default, that's why I went with these low numbers.